src/components/groups/group.middlewares.ts

```diff
@@ -23,6 +23,8 @@ export const joinGroup = asyncWrapper(async (req: Request, res: Response, next:
     sendMessage(res, 'Már tagja vagy ennek a csoportnak!')
   } else if ((group.users?.length || 0) >= group.maxAttendees) {
     sendMessage(res, 'Ez a csoport már tele van!')
+  } else if (group.endDate < new Date()) {
+    sendMessage(res, 'Ez a csoport már lejárt!')
```

Comment:
I'd change the phrasing a little bit, what do you think of 'Ez a csoport már véget ért!'?